pandas/core/algorithms.py

```diff
@@ -508,6 +512,7 @@ def duplicated(values, keep='first'):
     duplicated : ndarray
     """
 
+    values = com._asarray_tuplesafe(values)
```

Comment:
Your comment <a href="https://github.com/pandas-dev/pandas/pull/14934#discussion_r93349200">below</a> answers this question.  Fair enough.  Will remove.